core/dbt/include/global_project/macros/adapters/apply_grants.sql

```diff
@@ -0,0 +1,52 @@
+{% macro get_show_grant_sql(relation) %}
+{{ return(adapter.dispatch("get_show_grant_sql", "dbt")(relation)) }}
+{% endmacro %}
+
+{% macro default__get_show_grant_sql(relation) %}
+show grants on {{ relation.type }} {{ relation }}
+{% endmacro %}
+
+{% macro get_grant_sql(relation, grant_config) %}
+{{ return(adapter.dispatch('get_grant_sql', 'dbt')(relation, grant_config)) }}
+{% endmacro %}
+
+{% macro default__get_grant_sql(relation, grant_config) %}
+    {% for privilege in grant_config.keys() %}
+     {{ log('privilege: ' ~ privilege) }}
+        {% set grantees = grant_config[privilege] %}
+        {% for grantee in grantees %}
+            grant {{ privilege }} on {{ relation.type }} {{ relation }} to {{ grantee }};
```

Comment:
This works totally fine, just an ergonomic piece: It's possible to `grant <privilege> on <object> to first_grantee, second_grantee, third_grantee, ...`. It's slightly neater and avoids an extra looping step for us:
```sql
        {% set grantees = grant_config[privilege] %}
        grant {{ privilege }} on {{ relation.type }} {{ relation }} to {{ grantees | join(', ') }};
```